src/main/src/main/java/org/geoserver/catalog/LayerInfo.java

```diff
@@ -168,4 +168,28 @@
      */
     void setAdvertised(boolean advertised);
 
+    /**
+     * The default interpolation method.
+     *
+     * <p>
+     * If not specifed (i.e. {@code null}), the service default will be used.
+     * </p>
+     */
+    String getDefaultInterpolationMethod();
```

Comment:
This method applies to WMS only, it's not a default interpolation in WCS for example, better be explicit about it.
Also, since it can use only three values, best to use an enumeration, like WMSInfo.WMSInterpolation.
Hopefully this will also make it throw an exception is someone sends an invalid value over REST